client/src/telemetry/glean-context.tsx

```diff
@@ -174,8 +174,12 @@ function handleLinkClick(ev: MouseEvent, click: (source: string) => void) {
   if (anchor instanceof HTMLAnchorElement) {
     if (anchor.dataset.glean) {
       click(anchor.dataset.glean);
-    } else if (anchor.classList.contains("external")) {
-      click(`external-link: ${anchor.getAttribute("href") || ""}`);
+    }
+    if (anchor.href) {
+      const url = new URL(anchor.href, document.location.href);
```

Comment:
What are the chances of `anchor.href` containing an invalid URL, triggering an error?

Reply:
Can't wait for `URL.parse` to be baseline green. But yes let's add a `try...catch`